CHANGELOG

```diff
@@ -12,7 +12,7 @@
  * removed Twig_Autoloader (use Composer instead)
  * removed `true` as an equivalent to `html` for the auto-escaping strategy
  * removed pre-1.8 autoescape tag syntax
- * dropped support for PHP 5.2, 5.3, 5.4, and 5.5
+ * dropped support for PHP 5.x
```

Comment:
No, I've just removed HHVM from the Travis matrix, but I've not removed support for HHVM in the code. Actually, as HHVM should be compatible with PHP, we could even not care about HHVM at all as if something does not work, that's probably a bug on HHVM side.